tests/test/faaslet/test_filesystem.cpp

```diff
@@ -65,34 +65,55 @@ TEST_CASE_METHOD(FilesystemTestFixture, "Test listdir", "[faaslet]")
     execFunction(msg);
 }
 
-TEST_CASE_METHOD(FilesystemTestFixture, "Test fcntl", "[faaslet]")
+TEST_CASE_METHOD(FilesystemTestFixture, "Test fcntl", "[faaslet][wamr]")
 {
     faabric::Message msg = faabric::util::messageFactory("demo", "fcntl");
-    execFunction(msg);
+
+    SECTION("WAVM") { execFunction(msg); }
+
+    SECTION("WAMR") { execWamrFunction(msg); }
 }
 
+// TODO - bug in WAMR's WASI implementation makes this test fail. Will uncomment
+// when fixed.
 TEST_CASE_METHOD(FilesystemTestFixture, "Test fread", "[faaslet]")
 {
     faabric::Message msg = faabric::util::messageFactory("demo", "fread");
-    execFunction(msg);
+
+    SECTION("WAVM") { execFunction(msg); }
+
+    // SECTION("WAMR") { execWamrFunction(msg); }
```

Comment:
Ah, this relates to the `not working` comment earlier. This is the right place to have the comment and `TODO`, we can remove the `not working` comment from the Python module as that _is_ working.